cmd/cluster-init/main.go

```diff
@@ -56,7 +58,7 @@ func parseOptions() (options, error) {
 
 func validateOptions(o options) []error {
 	var errs []error
-	if o.clusterName == "" {
+	if !o.update && o.clusterName == "" {
```

Comment:
What do we do when `o.update && o.clusterName==""`?

Reply:
yes, all build clusters in the new `clusters/build-clusters/cluster-init.yaml` file get updated to current spec.